libafl_frida/src/helper.rs

```diff
@@ -836,6 +836,7 @@ impl<'a> FridaInstrumentationHelper<'a> {
         ));
     }
 
+    #[cfg(target_arch = "x86_64")]
```

Comment:
i have fixed this :)

Reply:
at line 842

Reply:
I must have missed it... Thanks!

Reply:
so can I merge this pr @tokatoka or should we fix something?

Reply:
Ah guess we can drop this line?

Reply:
Did it get duplicated?

Reply:
Yes. Fixing it right now

Reply:
Thanks. Sorry.